src/test/java/hw3/ex1/HomePageTest.java

```diff
@@ -0,0 +1,97 @@
+package hw3.ex1;
+
+import hw3.AbstractBaseSeleniumTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import java.util.*;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class HomePageTest extends AbstractBaseSeleniumTest {
+
+    private HomePageSteps steps;
+    private List<String> menuElements = new ArrayList<>(
+            Arrays.asList("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"));
+    private Map<String, String> benefitIcons = new HashMap<>();
+
+    {
+        benefitIcons.put("icons-benefit icon-practise", "To include good practices\n" +
+                "and ideas from successful\n" +
+                "EPAM project");
+        benefitIcons.put("icons-benefit icon-custom", "To be flexible and\n" +
+                "customizable");
+        benefitIcons.put("icons-benefit icon-multi", "To be multiplatform");
+        benefitIcons.put("icons-benefit icon-base", "Already have good base\n" +
+                "(about 20 internal and\n" +
+                "some external projects),\n" +
+                "wish to get more…");
+    }
+
+    @BeforeMethod
+    @Override
+    public void setUp() {
+        super.setUp();
+        steps = new HomePageSteps(driver);
+    }
+
+    @Parameters({"url", "pageTitle", "login", "password", "name",
+            "mainHeader", "mainText", "subHeader", "subHeaderLink"})
+    @Test
+    public void homePageTest(String url, String pageTitle, String login, String password, String name,
+                             String mainHeader, String mainText, String subHeader, String subHeaderLink) {
```

Comment:
Why do you need such amount of the parameters?

Reply:
Because there was a requirement to not hard code test data, so I decided to export almost all checked parameters.